test-types/types.ts

```diff
@@ -0,0 +1,3 @@
+expect({}).toBeDisabled()
```

Comment:
Shouldn't `expect` expect an element object here?

Reply:
nope, it expects anything